src/test/cypress/e2e/exercises/programming/ProgrammingExerciseParticipation.cy.ts

```diff
@@ -11,11 +11,15 @@ import {
 } from '../../../support/pageobjects/exercises/programming/OnlineEditorPage';
 import { convertCourseAfterMultiPart } from '../../../support/requests/CourseManagementRequests';
 
-// The user management object
+// Users
 const users = artemis.users;
+const admin = users.getAdmin();
+const studentOne = users.getStudentOne();
+const studentTwo = users.getStudentTwo();
+const studentThree = users.getStudentThree();
 
 // Requests
-const courseManagement = artemis.requests.courseManagement;
+const courseManagementRequests = artemis.requests.courseManagement;
```

Comment:
Not sure if maybe `courseManagementRequest` might be better, as I guess it's just one request?

Reply:
Good point, adjusted in the latest commit 👍